RealmSwift.podspec

```diff
@@ -0,0 +1,31 @@
+Pod::Spec.new do |s|
+  s.name                    = 'RealmSwift'
+  s.version                 = `sh build.sh get-version`
+  s.summary                 = 'Realm is a modern data framework & database for iOS & OSX.'
+  s.description             = <<-DESC
+                              Realm is a modern data framework & database for iOS & OSX. You can use it purely in memory — or persist to disk with extraordinary performance.
+
+                              Realm’s data structures look like NSObjects and NSArrays, but provide additional features such as: querying, relationships & graphs, thread safety, and more.
```

Comment:
That description was never updated since the original public release of realm-cocoa and should be reworded. I'll work with @timanglade on an update for both podspecs.

We'll also want to mention both podpspecs in each description and say when one should be used over the other.